src/modules/commander/commander.cpp

```diff
@@ -1341,6 +1341,11 @@ int commander_thread_main(int argc, char *argv[])
 	status_flags.circuit_breaker_engaged_gpsfailure_check = false;
 	get_circuit_breaker_params();
 
+	// initialize gps failure to false if circuit breaker enabled
+	if (status_flags.circuit_breaker_engaged_gpsfailure_check) {
+		status_flags.gps_failure = false;
+	}
```

Comment:
I would prefer if this line was somehow together with:
https://github.com/PX4/Firmware/blob/d64c459b1b64cf35b501b43dc527ed430bad1f02/src/modules/commander/commander.cpp#L1318

What about making it an `else` case?